lib/scripts/common/mesosflexinstall

```diff
@@ -51,10 +53,12 @@ function install_mesos {
 function configure_slave {
   if [[ ${slave_hostname+isset} ]]
   then
-    mkdir -p /etc/mesos-slave
+    mkdir -p /etc/mesos-slave/resources
     echo "$slave_hostname" > /etc/mesos-slave/hostname
     echo "docker,mesos" > /etc/mesos-slave/containerizers
     echo "5mins" > /etc/mesos-slave/executor_registration_timeout
+    echo "$slave_cpus" > /etc/mesos-slave/resources/cpus
+    echo "$slave_mem" > /etc/mesos-slave/resources/mem
```

Comment:
This seems unnecessary... @macmacbr can you provide some more context here as to why this needs to be configurable? By default the slave process will use all the CPU/MEM available inside the VM.